build.gradle

```diff
@@ -51,8 +52,7 @@ def today = Instant.now()
 def todaysDate = LocalDate.now().format('yyyy-MM-dd')
 version = System.getenv('CI') ? "${todaysDate}.${System.getenv('CIRCLE_BUILD_NUM')}" : todaysDate
 
-jar {
-    baseName = rootProject.name
+bootJar {
```

Comment:
`jar` task isn't called by spring boot.  The `bootJar` task will add the version so that it is then displayed on startup